test/app-luatest/protobuf_I32_test.lua

```diff
@@ -0,0 +1,276 @@
+local t = require('luatest')
```

Comment:
Completed tests parametrization for numeric protobuf types using luatest parametrization. I suspect that `LEN` tests will stay out of parametrization same as `module` and `repeated` because they check specific cases. Anyway I suppose it will be better if you check this changes once again before merging.

Reply:
I glanced over the numeric test and the approach seems good. I also agree that parametrization is not applicable for all the test cases, but is good for numeric ones. Thanks for the update!